src/transform/FooterContainer.js

```diff
@@ -0,0 +1,92 @@
+import './FooterContainer.css'
+
+/**
+ * Ensures the 'Read more' section header can always be scrolled to the top of the screen.
+ * @param {!Document} document
+ * @param {!Window} window
+ * @return {void}
+ */
+const updateBottomPaddingToAllowReadMoreToScrollToTop = (document, window) => {
+  const div = document.getElementById('pagelib_footer_container_ensure_can_scroll_to_top')
```

Comment:
Ok to hold off on this for now? The template complicates doing this... it's filled with such ids...